manifests/fragment.pp

```diff
@@ -68,6 +68,7 @@
 
   $safe_name        = regsubst($name, '[/:\n]', '_', 'GM')
   $safe_target_name = regsubst($target, '[/:\n]', '_', 'GM')
+  $safe_order       = regsubst($order, '[/:\n]', '_', 'GM')
```

Comment:
Depends on what you mean by work.

According to the documentation, the order parameter is a string regardless of whether sorting is alpha or numeric. The function will replace the 3 characters in the regexp with underscores. None of those 3 characters are valid in a number. So, in some sense, this will let you replace a broken non-number with a different broken non-number that happens to be more filesystem-safe.

I have to admit it's not clear to me why the module differentiates between alpha sorting and numerical sorting when they are both instances of lexical sorting.